### PR TITLE
Portfolio page and hotfixes

### PR DESCRIPTION
**Pull request by sreen020**
# Description

Closes #

_[Please include a summary of the change and which issue is fixed]_

## Type of change

Please select the option that is relevant

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

# Checklist

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] All the required documentation is added.
- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.

# Additional information

_[Provided extra information if necessary]_

### CODE REVIEW

File: src/app.css [hunk 1]
```diff
@@ -102,6 +102,7 @@ section > h1 {
 	background: url('$lib/assets/images/background/leavesBackground.svg');
 	background-position: 20% 0;
 	background-size: 250%;
+	height: 100%;
```
Comment (jochemvogel) [inline src/app.css:105]: Weet je zeker 🤣

**Comment by jochemvogel** [comment]
Is het idee om die mockup erin te doen?

File: src/routes/portfolio/index.svelte [hunk 2]
```diff
@@ -0,0 +1,85 @@
+<main>
```
Comment (iamDistractech) [inline src/routes/portfolio/index.svelte:5]: Als je dit erbij doet, wordt de pagina al geprerenderd tijdens de build. Dat is wat beter omdat dit een statische pagina is:
```svelte
<script context="module">
	export const prerender = true;
</script>
```

Comment (iamDistractech) [reply src/routes/portfolio/index.svelte:5]: Je kan dan in de service worker ook dit aanvullen: 

`const ssrPages = ['/', '/spellen/offline', '/gymles', '/spellen', '/inloggen', /portfolio];`

**Comment by jochemvogel** [comment]
Merge dan

**Review by jochemvogel — APPROVED** [review]

**Review by iamDistractech — APPROVED** [review]

**Pull request merged**